src/main/java/pwr/asi/java/hermes/entities/Mail.java

```diff
@@ -0,0 +1,79 @@
+package pwr.asi.java.hermes.entities;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.Objects;
+
+@Entity
+@Table(name = "mails")
+@SecondaryTable(name = "mail_counter")
```

Comment:
JPA tend to keep single class per entity (or even a part of entity as in `@MappedSuperclass`). There are other ways we can do it and I would like to discuss them. But don't get me wrong - it is not a bad solution! :wink: